From 94b97065a30a51aeaa7481e843d288b177c87db9 Mon Sep 17 00:00:00 2001
Message-Id: <94b97065a30a51aeaa7481e843d288b177c87db9@dist-git>
From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Mon, 7 Apr 2014 13:30:43 +0200
Subject: [PATCH] libvirt: fix error message when connection can't be opened

VIR_ERR_NO_CONNECT already contains "no connection driver available".

This patch changes:
no connection driver available for No connection for URI hello
to:
no connection driver available for hello

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=851413
(cherry picked from commit 633d859b7375ff33c8307addf92683bc05ac8798)

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/libvirt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libvirt.c b/src/libvirt.c
index e8ccdf8..09936d4 100644
--- a/src/libvirt.c
+++ b/src/libvirt.c
@@ -1188,7 +1188,7 @@ do_open (const char *name,
     if (!ret->driver) {
         /* If we reach here, then all drivers declined the connection. */
         virLibConnError(VIR_ERR_NO_CONNECT,
-                        _("No connection for URI %s"),
+                        "%s",
                         NULLSTR(name));
         goto failed;
     }
-- 
1.9.1

